pycbc/io/hdf.py

```diff
@@ -448,6 +449,22 @@ def __init__(self, trig_file, bank_file, veto_file,
             logging.info('%i triggers remain after cut on %s',
                          sum(self.mask), filter_func)
 
+    def __getitem__(self, key):
+        # Is key in the TRIGGER_MERGE file?
+        try:
+            vals = self.get_column(key)
+            return vals
```

Comment:
Looks like you can save both a line and a variable here.

Reply:
Thanks, changed.